packages/cyberstorm/src/components/Tag.tsx

```diff
@@ -6,19 +6,20 @@ export interface TagProps {
   leftIcon?: ReactNode;
   rightIcon?: ReactNode;
   tagStyle?: "tag__default";
+  tagSize?: "tag__small" | "tag__medium" | "tag__large";
```

Comment:
I think the name of the prop should be just `size`, since what else could it refer to than the component itself? In comparison, I assumed `tagStyle` has the tag-prefix to avoid confusing it with HTML's `style` attribute.

Secondly, I feel that having the `tag__` prefix on the prop value is unnecessarily verbose. Something like `size="small"` is what I'd expect, if I was a developer using such a component. So maybe the prop value should be decoupled from the class name, letting the component handle forming the class name internally?

I guess both of these tie into the naming convention thing, and perhaps they aren't blockers for this PR necessarily. But we should have that discussion (again) sometime soon? Could you ensure that the things mentioned here are included in the related ticket?